docusaurus/docs/scheduler/scheduler-getting-started.md

```diff
@@ -60,6 +60,8 @@ services:
     working_dir: /app/build/services/scheduler
     command: "api"
     hostname: scheduler
+    volumes:
+      - ./init-data-volume/:/app/services/scheduler/init-data-volume
```

Comment:
This should also be documented in docs similar to how it is in mailer - https://thesoftwarehouse.github.io/rad-modules-docs/docs/mailer/mailer-advanced-configuration production usage case